src/material-experimental/mdc-slider/slider.ts

```diff
@@ -48,6 +55,173 @@ export interface MatSliderDragEvent {
   value: number;
 }
 
+/**
+ * The visual slider thumb.
+ *
+ * Handles the slider thumb ripple states (hover, focus, and active),
+ * and displaying the value tooltip on discrete sliders.
+ */
+@Component({
+  selector: 'mat-slider-visual-start-thumb, mat-slider-visual-end-thumb',
+  templateUrl: './slider-thumb.html',
+  host: {
+    'class': 'mdc-slider__thumb',
+    '(mouseenter)': '_onMouseEnter()',
+    '(mouseleave)': '_onMouseLeave()',
+  },
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
+})
+export class MatSliderVisualThumb implements AfterViewInit {
+  /** Whether the slider displays a numeric value label upon pressing the thumb. */
+  @Input() _discrete: boolean;
+
+  /** The display value of the slider thumb. */
+  @Input() _valueIndicatorText: string;
+
+  /** The MatRipple for this slider thumb. */
+  @ViewChild(MatRipple) private readonly _ripple: MatRipple;
+
+  /** The slider thumb knob */
+  @ViewChild('knob') _knob: ElementRef<HTMLElement>;
+
+  /** Indicates which slider thumb this input corresponds to. */
+  _thumbPosition: Thumb =
+    this._elementRef.nativeElement.tagName.toLowerCase()
+      === 'mat-slider-visual-start-thumb'.toLowerCase()
+        ? Thumb.START
+        : Thumb.END;
+
+  /** The slider input corresponding to this slider thumb. */
+  private _sliderInput: MatSliderThumb;
+
+  /** The RippleRef for the slider thumbs hover state. */
+  private _hoverRippleRef: RippleRef;
+
+  /** The RippleRef for the slider thumbs focus state. */
+  private _focusRippleRef: RippleRef;
+
+  /** The RippleRef for the slider thumbs active state. */
+  private _activeRippleRef: RippleRef;
+
+  /** Whether the slider thumb is currently being pressed. */
+  private _isActive: boolean = false;
+
+  /** Whether the slider thumb is currently being hovered. */
+  private _isHovered: boolean = false;
+
+  constructor(
+    private readonly _slider: MatSlider,
+    private readonly _elementRef: ElementRef<HTMLElement>) {}
+
+  ngAfterViewInit() {
+    this._ripple.radius = 24;
```

Comment:
This may be an issue with different densities - is there a way to not set this and let the ripple renderer figure it out on the fly? Maybe check out the radio button does it

Reply:
I'm not sure what you mean here. I checked out [our mdc-radio](https://github.com/angular/components/blob/master/src/material-experimental/mdc-radio/radio.ts#L64) and we are only changing the `enterDuration` and `exitDuration` for that ripple

While on this topic, though, reading through radio.ts, I saw we have a `disableRipple` input
https://github.com/angular/components/blob/master/src/material-experimental/mdc-radio/radio.ts#L110

Is this something the slider should have, too?

Reply:
Try to change the density of the slider and see if the ripple still makes sense. I suspect it should get smaller but isn't since you hard-coded that the slider if 48px. 

See what happens if you don't set the radius - it should automatically determine the radius on the fly if its not set

Reply:
I tried changing the density but the slider didn't seem to change at all. Is there something wrong with the way it's set up in [_slider-theme.scss](https://github.com/angular/components/blob/15c0a31879818dfa490549f41594e7618c9f221c/src/material-experimental/mdc-slider/_slider-theme.scss)?

Reply:
Hmm no - looks like you got it right. They haven't appeared to have implemented it yet